core/server/api/v2/themes.js

```diff
@@ -62,6 +62,8 @@ module.exports = {
 
             return themeService.storage.setFromZip(zip)
                 .then((theme) => {
+                    // CASE: clear cache
+                    this.headers.cacheInvalidate = true;
```

Comment:
In "frame" we can set headers cache invalidation in the configuration of the controller, like in `activate` method: https://github.com/TryGhost/Ghost/pull/10894/files#diff-46b48a682fac349fca2e4060e8030927R17 .
The `setFromZip` method was also extracted from API v0.1, so would need an update there

Reply:
Nevermind, just read the comment in the PR :smile: